packs/equipment/barding-of-the-zephyr.json

```
@@ -1,16 +1,21 @@
 {
-    "_id": "eoI3M6FXtcPWeg7i",
+    "_id": "MKVYBJ2Uk5efL7ku",
```

```suggestion
    "_id": "eoI3M6FXtcPWeg7i",
```
We should preserve the IDs when possible.